examples/globe.js

```diff
@@ -3,18 +3,48 @@
 
 // Define initial camera position
 var positionOnGlobe = { longitude: 2.351323, latitude: 48.856712, altitude: 25000000 };
+var promises = [];
+var miniView;
+var minDistance = 10000000;
+var maxDistance = 30000000;
 
 // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
 var viewerDiv = document.getElementById('viewerDiv');
+var miniDiv = document.getElementById('miniDiv');
 
 // Instanciate iTowns GlobeView*
 var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer: renderer });
-
-var promises = [];
-
 function addLayerCb(layer) {
     return globeView.addLayer(layer);
 }
+
+if (!renderer) {
+    // Instanciate new Globe viewer to add mini view
+    // maxSubdivisionLevel limits display level to zoom/level at 2
+    // alphaClearValue at 0 sets opacity rendering to totaly transparent
```

Comment:
The 2 lines could be replaced by something explaining the intent for each parameter:

```js
miniView = new itowns.GlobeView(miniDiv, positionOnGlobe, {
        renderer: renderer,
        // `limit globe' subdivision level: we're don't need a precise globe model since the mini globe will always be seen from a far point of view (see minDistance above)
        maxSubdivisionLevel: 2,
        // don't create default controls since miniview's camera will be synced on the main view's one
        noControls: true,
        // force a 0 alpha clear value (instead of the default '1') because we want a transparent background for the miniglobe view to be able to see the main view "behind"
       alphaClearValue: 0
    });
```